### PR TITLE
Fixed preview to save before render and avoid over writing cached values

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [x] test coverage

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| Related PRs      | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Component/Content/Compat/Serializer/PageBridgeHandler.php [hunk 1]
```diff
@@ -60,54 +58,61 @@ public static function getSubscribingMethods()
...
-        $structureProperty->setAccessible(true);
+        $document = $bridge->getDocument();
+        $structure = $bridge->getStructure();
 
-        $document = $documentProperty->getValue($bridge);
-        $structure = $structureProperty->getValue($bridge);
+        $data = [];
+        foreach ($bridge->getProperties(true) as $property) {
+            $data[$property->getName()] = $property->getValue();
+        }
```
Comment (danrot) [inline src/Sulu/Component/Content/Compat/Serializer/PageBridgeHandler.php]: Why is this necessary? Shouldn't the document also have this information? Then I would try to implement that correctly in the document serialization, since the PageBridge will be gone in the long term.

Comment (wachterjohannes) [inline src/Sulu/Component/Content/Compat/Serializer/PageBridgeHandler.php]: the best way would be to serialize only the data and set it back to the newly generated document. then we would not have any problems at all!

File: CHANGELOG.md [hunk 2]
```diff
@@ -191,9 +193,6 @@ CHANGELOG for Sulu
     * ENHANCEMENT #1342 [SecurityBundle]  Added creator and changer of contact to 'fullContact' serialization group
     * BUGFIX      #1365 [ContactBundle]   Fixed bug caused by new instance name of datagrid
     * BUGFIX      #1136 [MediaBundle]     Fixed image scale forceRatio parameter for none squared image formats
-
-* dev-master
```
Comment (wachterjohannes) [inline CHANGELOG.md]: will no be released in a hotfix!

File: src/Sulu/Bundle/ContentBundle/Preview/Preview.php [hunk 3]
```diff
@@ -175,20 +171,22 @@ private function update(
...
-                    [$property => $changes],
-                    $userId,
-                    $content->getUuid(),
-                    $webspaceKey,
-                    $locale
-                );
-            }
+        } catch (\Twig_Error $ex) {
+            // rendering error
+            $changes = [];
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Preview/Preview.php]: Should we just ignore this? I think it should at least be logged, if not even show an error label in the administration. @chirimoya what do you think?

Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Preview/Preview.php]: sorry i have overseen this line:

```throw new TwigPreviewException($ex);```

i will add it here!
